src/controllers/request.controller.js

```diff
@@ -3,14 +3,25 @@ import responseHandlers from '../utils/responseHandlers';
 import statusCodes from '../utils/statusCodes';
 import customMessages from '../utils/customMessages';
 import RequestService from '../services/request.service';
+import Validators from '../utils/validators';
+import userRoles from '../utils/userRoles.util';
 
-const { handleTripRequest, getMostTraveledDestinations, acceptRequest } = RequestService;
+const { 
+  handleTripRequest, 
+  getMostTraveledDestinations,
+  acceptRequest,
```

Comment:
I think this accept shouldn't be here, since the acceptRequest will be another story